### PR TITLE
Orm

### PR DESCRIPTION
**Pull request by whiterook6**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[*]
+end_of_line = lf
+indent_size = 2
```
Comment (whiterook6) [inline .editorconfig:3]: urgh, fine.

File: lib/chat.js [hunk 2]
```diff
@@ -28,25 +17,6 @@ var Chat = {
 
 	send_team: function(player, to_team, text){
 		Logger.debug('Chat.send_team');
-		if (!player.initials){
```
Comment (whiterook6) [inline lib/chat.js:31]: Isn't this the kind of stuff you should be warning in the log? Or are you relying on the exceptions?

File: .editorconfig (hunk 1 above)
Comment (mayppong) [inline .editorconfig:3]: This only kicks in if you enable editorconfig plugin in your editor.

File: lib/chat.js (hunk 2 above)
Comment (mayppong) [inline lib/chat.js:31]: We're already checking it before we call this function.

**Pull request merged**